ares/ares/node/debugger/tracer/instruction.hpp

```diff
@@ -57,7 +57,7 @@ struct Instruction : Tracer {
   //call when writing to executable RAM to support self-modifying code.
   auto invalidate(u64 address) -> void {
     if(unlikely(_mask && updateMasks())) {
-      address &= (1ull << _addressBits) - 1;
+      address &= (u128(1) << _addressBits) - 1;
       address >>= _addressMask;
       _masks[address >> 3] &= ~(1 << (address & 7));
     }
```

Comment:
GitHub apparently doesn't allow comments on lines not included in a change, but line 107 appears broken as well. It should be replaced with something like this:
```
auto size = 1ull << (_addressBits - _addressMask - 3);
```